Source/Scene/Cesium3DTile.js

```diff
@@ -32,6 +32,7 @@ import Cesium3DTileRefine from "./Cesium3DTileRefine.js";
 import Empty3DTileContent from "./Empty3DTileContent.js";
 import findGroupMetadata from "./findGroupMetadata.js";
 import has3DTilesExtension from "./has3DTilesExtension.js";
+import TileMetadata from "./TileMetadata.js";
```

Comment:
Move into alphabetic order. I'm guessing this used to be called `MetadataTile`.

Reply:
yup exactly, after the rename in the other PR I did a search and replace.